app/services/sockethub-irc.js

```diff
@@ -0,0 +1,180 @@
+import Ember from 'ember';
+
+const {
+  inject: {
+    service
+  }
+} = Ember;
+
+/**
+ * This service provides helpers for SocketHub IRC communications
+ * @module hyperchannel/services/sockethub-irc
+ */
+export default Ember.Service.extend({
+
+  logger: service(),
+  coms: service(),
```

Comment:
This service's unit test actually makes sure it has access to coms. When we move more functions here it'll be needed I think.